diff -crB apache-log4cxx-0.10.0/src/examples/cpp/console.cpp installed_packages/apache-log4cxx-0.10.0/src/examples/cpp/console.cpp
*** apache-log4cxx-0.10.0/src/examples/cpp/console.cpp	2008-03-31 18:34:52.000000000 +0000
--- installed_packages/apache-log4cxx-0.10.0/src/examples/cpp/console.cpp	2013-03-04 17:00:01.355823009 +0000
***************
*** 16,21 ****
--- 16,23 ----
   */
  
  #include <stdlib.h>
+ #include <string.h>
+ #include <stdio.h>
  #include <log4cxx/logger.h>
  #include <log4cxx/consoleappender.h>
  #include <log4cxx/simplelayout.h>
